clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -706,10 +706,10 @@ private KafkaConsumer(ConsumerConfig config,
                   Metrics metrics,
                   SubscriptionState subscriptions,
                   Metadata metadata,
-                  boolean autoCommitEnabled,
                   int autoCommitIntervalMs,
                   int heartbeatIntervalMs,
                   long retryBackoffMs,
+                  boolean autoCommitEnabled,
                   long requestTimeoutMs) {
```

Comment:
True. I went ahead and did that in KAFKA-3807. I'm not sure which will land first, but feel free to do it here.